css-fonts-4/Overview.bs

```diff
@@ -6585,7 +6585,7 @@ None.
 <h3 id="sp213">How does this specification distinguish between behavior in first-party and third-party contexts?</h3>
 
 <a href="#font-fetching-requirements">For font loads, user agents must use the
-potentially CORS-enabled fetch method defined by the [[!HTML]] specification
```

Comment:
No, that prevents HTML being a normative reference.
I agree tat the nested link should be fixed, but not like that.

Reply:
Fixed, I restored the biblio link and just pulled back the `a` to only cover the starting phrase, which is still a reasonable link text I think.